.changes/unreleased/Breaking Changes-20220923-112314.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Breaking Changes
+body: Add timestamps.sql to consolidate timestamp macros
+time: 2022-09-23T11:23:14.312738-07:00
+custom:
+  Author: colin-rogers-dbt
+  Issue: "5521"
+  PR: "273"
```

Comment:
I'm not sure exactly how the changie YAML files work, but I'm _guessing_ that this will look for issue number 5521 within dbt-snowflake rather than 5521 within dbt-core.